projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1872,6 +1888,20 @@ protected static function build_sprintf( $args ) {
 	 * @return string
 	 */
 	public function publicize_connections_url( $source = 'calypso-marketing-connections' ) {
+
+		$is_simple_site = defined( 'IS_WPCOM' ) && IS_WPCOM;
+		$is_atomic_site = ( new Status\Host() )->is_woa_site();
+
+		$is_wpcom = $is_simple_site || $is_atomic_site;
```

Comment:
Should this be part of the `use_admin_ui_v1` check? We're essentially only wanting to use the new URLs if the feature is enabled and it's not on WPCOM

Reply:
I think we should rename `use_admin_ui_v1` to something `use_calypso_ui` and use that to decide to show the appropriate UI for WPCOM and Jetpack sites.

Also, we use `use_admin_ui_v1` for the endpoints and schema structure, which means that the editor in WPCOM sites still uses the old schema. Moving forward, we should use the same schema for all the data for all the sites. Having a flag like `use_calypso_ui` will allow us to have the same schema but different UIs for WPCOM and Jetpack sites.